Q/c_questdb_client/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+using BinaryBuilder
+using BinaryBuilderBase
+
+name = "c_questdb_client"
+version = v"2.1.3"
+
+sources = [    
+    GitSource("https://github.com/questdb/c-questdb-client", "ad3776efb057d09a86a83e15c0f39ae40d75485b")
+]
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/c-questdb-client/questdb-rs-ffi
+cargo build --release
+if [[ "${target}" == *-w64-mingw32* ]]; then    
+    install -D -m 755 "target/${rust_target}/release/questdb_client.${dlext}" "${libdir}/c_questdb_client.${dlext}"
+else
+    install -D -m 755 "target/${rust_target}/release/libquestdb_client.${dlext}" "${libdir}/c_questdb_client.${dlext}"
+fi
```

Comment:
I'd simplify to
```suggestion
install -D -m 755 "target/${rust_target}/release/*questdb_client.${dlext}" "${libdir}/libquestdb_client.${dlext}"
```
and call the product below `libquestdb_client`